packages/build-utils/test/integration.test.ts

```diff
@@ -221,51 +220,3 @@ it('Test `detectBuilders` with `index` files', async () => {
   );
   expect(deployment).toBeDefined();
 });
-
-it("Should cache pnpm Plug'N'Play with symlink=false", async () => {
-  const fixture = path.join(__dirname, 'fixtures', '24-pnpm-pnp-symlink');
-
-  const config1 = {
-    probes: [
-      {
-        path: '/',
-        mustContain: 'Hello, World',
-        logsMustContain: 'Build Cache not found',
-      },
-    ],
-  };
-
-  await fs.writeFile(
-    path.join(fixture, 'vercel.json'),
-    JSON.stringify(config1, null, 2)
-  );
-
-  const deployment1 = await testDeployment(
-    { builderUrl, buildUtilsUrl },
-    fixture
-  );
-
-  expect(deployment1).toBeDefined();
-
-  const config2 = {
-    probes: [
-      {
-        path: '/',
-        mustContain: 'Hello, World',
-        logsMustContain: 'Build cache downloaded',
-      },
-    ],
-  };
-
-  await fs.writeFile(
-    path.join(fixture, 'vercel.json'),
-    JSON.stringify(config2, null, 2)
-  );
-
-  const deployment2 = await testDeployment(
-    { builderUrl, buildUtilsUrl },
-    fixture
```

Comment:
We would probably need to preserve the projectId from the previous deployment to actually test the build cache.